openapi.yaml

```diff
@@ -3497,8 +3500,739 @@ paths:
           $ref: '#/components/responses/client_error_auth'
         5XX:
           $ref: '#/components/responses/server_error'
+  '/commercial_data/collections/{collection_id}':
+    get:
+      summary: Information about searching and ordering a commercial data collection
+      operationId: commercial-data-collection-info
+      description: |-
+        Lists information about pricing and available parameters for searching and ordering products for a commercial data collection specified by the identifier `collection_id`.
+        STAC-compliant general information about the collection must be available at `GET collections/{collection_id}`.
+      tags:
+        - Commercial Data
+      security:
+        - {}
+        - Bearer: []
+      parameters:
+        - name: collection_id
+          in: path
+          description: Collection identifier
+          required: true
+          schema:
+            $ref: '#/components/schemas/collection_id'
+      responses:
+        '200':
+          description: Information about commercial data collection.
+          content:
+            application/json:
+              schema:
+                title: Commercial Collection Info
+                type: object
+                required:
+                  - id
+                  - pricing
+                  - search_parameters
+                  - order_parameters
+                properties:
+                  id:
+                    type: string
+                    description: A unique identifier for the collection, which MUST match the specified pattern.
+                    pattern: '^[\w\-\.~\/]+$'
+                  provider:
```

Comment:
This should probably be aligned with STACs `providers` field.

Reply:
Good idea, done so.